script/Ciel/Day 6/06_01_CIEL06_6.txt

```diff
@@ -443,12 +443,14 @@ I admit that I am also a little curious about what you've been up to in the Nort
 [sha:4c23e3d58ec835120e0227fd94aa438bf6c48124]{
 -- Page 252, Offset 13823.
 -- さて。それでは貴方の中の原理を、わずかですが現実に<零|こぼ>す神秘を教授いたしましょう。
+// Make this not read so weird
 Now then. Allow me to teach you an arcanum for spilling――
 if only minutely—―
 the Principle you hold within you onto reality.
 }
 [sha:ed2ae5ed4dca523c35c9e287d935981668d26ec6]{
 -- Page 252, Offset 13824.
 -- その神代の御業をもって、貴方に捕えてほしいひとが―――』
+// Also reads extremely awkwardly
```

Comment:
I'll leave these to the lore council

Reply:
>Very well, I shall teach you an arcanum that'll let you spill the Principle you hold within onto reality, even if only by a little.
>There is someone I wish for you to capture with this instrumentality fostered in the Age of Gods.

Uh, maybe this is a bit better?

Reply:
Would probably prefer 'even if only slightly' over 'by a little' just because it feels weird to spill something _by_ a little (instead of spilling a little _of_ something).
Also, `Instrumentality` has pretty high obtuse factor for 'a thing you can use' imo. Is it physical or virtual? Can it be e.g. `artifact` or `technique`?

Reply:
It's some sort of technique/spell/whatever, I just tried to keep the religious dog-shit still in there somehow.

Reply:
I don't really get religious from instrumentality - divine rite / ritual that dates back to the age of gods?

Reply:
>There is someone I wish for you to capture with this rite from the Age of Gods.

Yeah, good idea. 'Dates back' can even be simplified to 'from', which works well.